basedtyping/__init__.py

```diff
@@ -183,19 +199,26 @@ def __class_getitem__(cls, item: T) -> type[ReifiedGeneric[T]]:
 
 
 # TODO: make this work with any "form", not just unions
-def issubform(type1: type | UnionType, type2: type | UnionType) -> bool:
-    """``issubclass`` but works with a union as the first arg (like it does for the second arg)
+#  should be (form: TypeForm, forminfo: TypeForm)
+def issubform(form: type | UnionType, forminfo: type | UnionType) -> bool:
+    """Returns ``True`` if ``form`` is a subform (specialform or subclass) of ``forminfo``.
+
+    Like  ``issubclass`` but supports typeforms (type-time types)
```

Comment:
i wouldn't add this part until it actually supports more than one typeform

Reply:
I added a big warning